q2_cutadapt/plugin_setup.py

```diff
@@ -32,6 +45,150 @@
                   'pp. 10-12.\ndoi:http://dx.doi.org/10.14806/ej.17.1.200',
 )
 
+plugin.methods.register_function(
+    function=q2_cutadapt._trim.trim_single,
+    inputs={
+        'demultiplexed_sequences': SampleData[SequencesWithQuality],
+    },
+    parameters={
+        'cores': Int % Range(1, None),
+        'adapter': List[Str],
+        'front': List[Str],
+        'anywhere': List[Str],
+        'error_rate': Float % Range(0, 1, inclusive_start=True,
+                                    inclusive_end=True),
+        'indels': Bool,
+        'times': Int % Range(1, None),
+        'overlap': Int % Range(1, None),
+        'match_read_wildcards': Bool,
+        'match_adapter_wildcards': Bool,
+    },
+    outputs=[
+        ('trimmed_sequences', SampleData[SequencesWithQuality]),
+    ],
+    input_descriptions={
+        'demultiplexed_sequences': 'The single-end sequences to be trimmed.',
+    },
+    parameter_descriptions={
+        'cores': 'Number of CPU cores to use.',
+        'adapter': 'Sequence of an adapter ligated to the 3\' end. The '
+                   'adapter and any subsequent bases are trimmed. If a `$` '
+                   'is appended, the adapter is only found if it is a suffix '
+                   'of the read.',
+        'front': 'Sequence of an adapter ligated to the 5\' end. The adapter '
+                 'and any preceding bases are trimmed. Partial matches at the '
+                 '5\' end are allowed. If a `^` character is prepended, the '
+                 'adapter is only found if it is a prefix of the read.',
```

Comment:
If you agree with my previous comment, you should do this too:

_a prefix of the read_ -> _at the beginning of the read_